sdk/js/src/token_bridge/getIsTransferCompleted.ts

```diff
@@ -64,6 +68,30 @@ export async function getIsTransferCompletedTerra(
   return false;
 }
 
+/**
+ * This function is used to check if a VAA has been redeemed on terra2 by
+ * querying the token bridge contract.
+ * @param tokenBridgeAddress The token bridge address (bech32)
+ * @param signedVAA The signed VAA byte array
+ * @param client The LCD client. Only used for querying, not transactions will
+ * be signed
+ */
+export async function getIsTransferCompletedTerra2(
```

Comment:
I need to change getIsTransferCompletedInjective() to look like this, right?

Reply:
Correct, once this PR is merged